data/json/recipes/armor/hands.json

```diff
@@ -827,6 +827,59 @@
     "time": "6 h 45 m",
     "using": [ [ "tailoring_wool_knitting", 140 ] ]
   },
+  {
+    "result": "gloves_wool_fingerless",
+    "id_suffix": "from_patchwork",
```

Comment:
You don't need an id_suffix for the "base recipe", it's used to differentiate multiple recipes for the same item.

```suggestion
```